### ISSUE TITLE
"Bundle build failed"

### ISSUE THREAD

**Issue by christianselig** [johnno1962/InjectionApp#31]
I keep getting this error in Injection's log when I try to inject with Xcode 9 beta 4:

```
Copying iOSBundleTemplate into project. usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file        
cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file … target_directory  

*** Could not copy injection bundle. ***   at /Applications/Injection.app/Contents/Resources/common.pm line 61. 	
main::error(‘Could not copy injection bundle.’) called at /Applications/Injection.app/Contents/Resources/injectSource.pl line 73 
*** Bundle build failed ***
```

It was working fine in Xcode 8 as far as I know, and that is with using the Xcode 9 version from Twitter.

**Comment by johnno1962** [comment]
Not sure what’s going on there at all. Have you tested a simple example project works? If I was to take a guess, something has a space in the filename somewhere which is not supported.

**Comment by christianselig** [comment]
@johnno1962 Yeah, with an example project (quite literally a bare "Single View App" project), under iOS 11 in the simulator the injection is logged as occurring, but `injected()` is never called in `ViewController`.

In the iOS 10 simulator I get an EXC_BAD_ACCESS from Xcode upon injection followed by "Timeout waiting for connection from client app" from the Injection app.
